### ISSUE TITLE
Browser process close errors when running more than 1 iteration

### ISSUE THREAD

**Issue by imiric** [grafana/xk6-browser#415]
Currently (on eb13672) the following command:

```
XK6_BROWSER_LOG=info XK6_HEADLESS=1 xk6 run -u 1 -i 5 examples/fillform.js
```

Produces these errors:

```
ERRO[0002] cmd.Wait: signal: killed                      category="BrowserType:execute" elapsed="0 ms" goroutine=22
ERRO[0002] stdout.Close: close |0: file already closed   category="BrowserType:execute" elapsed="0 ms" goroutine=22
```

It seems to happen on each iteration end.

**Comment by imiric** [comment]
This seems to be a recent regression that's not present in v0.3.0.

`git bisect` points to it being introduced in #409 (72b006d).

@ankur22 Could you take a look? We should fix this before releasing v0.4.0.

**Comment by ankur22** [comment]
> This seems to be a recent regression that's not present in v0.3.0.
> 
> `git bisect` points to it being introduced in #409 ([72b006d](https://github.com/grafana/xk6-browser/commit/72b006dfab0d698faff183b8ba8b41b5af340c73)).
> 
> @ankur22 Could you take a look? We should fix this before releasing v0.4.0.

Yep, the change (to log any errors during `cmd.Wait()` and `stdout.Close()`) has surfaced a couple of bugs:
1. `stdout.Close: close |0: file already closed` -- we don't need to call `stdout.Close()` since the `cmd.Wait()` will close the `stdout` pipe for us (Source: https://pkg.go.dev/os/exec#Cmd.StdoutPipe)
2. `cmd.Wait: signal: killed ` -- :
    1. Is it because a process is killing it (only thing i can find is if there is a panic, then it will get the PID of the browser process and kill it)
    2. Is it due to the context closing (Source: https://pkg.go.dev/os/exec#CommandContext). Unfortunately we can't unwrap the error and look for a context closed error so instead we could just do a simple `err.Error() != "signal: killed"` and only print errors if it's not been killed.

**Comment by inancgumus** [comment]
> 1. `stdout.Close: close |0: file already closed` -- we don't need to call `stdout.Close()` since the `cmd.Wait()` will close the `stdout` pipe for us (Source: https://pkg.go.dev/os/exec#Cmd.StdoutPipe)
> 2. `cmd.Wait: signal: killed ` --   It is due to the context closing (Source: https://pkg.go.dev/os/exec#CommandContext). Unfortunately we can't unwrap the error and look for a context closed error so instead we could just do a simple `err.Error() != "signal: killed"` and only print errors if it's not been killed.

That's why the previous code discarded them 🙄 I don't know about you, but I prefer discarding both errors. You might want to keep the second error, though.

```go
// Previously
_ = cmd.Wait()
_ = stdout.Close()
```
